src/cluster/backend/zeromq/ZeroMQ.cc

```diff
@@ -325,6 +352,10 @@ bool ZeroMQBackend::DoUnsubscribe(const std::string& topic_prefix) {
         // Prepend 0x00 byte to indicate subscription to XSUB socket.
         // This is the XSUB API instead of setsockopt(ZMQ_SUBSCRIBE).
         std::string msg = '\0' + topic_prefix;
+
+        // Send a two message parts. The first part is a single byte tagging the
```

Comment:
Should we say "unsubscription" everywhere in the body of this function where it currently says "subscription"?